### PR TITLE
Add net.xm1math.Texmaker

### PR DESCRIPTION
**Pull request by dgcampea**

### CODE REVIEW

File: net.xm1math.Texmaker.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.texlive
+command: texmaker
+rename-desktop-file: texmaker.desktop
+rename-appdata-file: texmaker.appdata.xml
+rename-icon: texmaker
+finish-args:
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=wayland
```
Comment (TingPing) [inline net.xm1math.Texmaker.yaml]: Just curious if tested on Wayland, common issues there for Qt.

Comment (dgcampea) [reply net.xm1math.Texmaker.yaml]: Not tested on Wayland. Should I remove it and change to socket=x11?

**Comment by LeandroStanger** [comment]
bot, build net.xm1math.Texmaker

**Comment by bilelmoussaoui** [comment]
Have you reached upstream if they were interested in (co-)maintaining their app on Flathub?

File: appdata.patch [hunk 2]
```diff
@@ -0,0 +1,37 @@
+--- a/utilities/texmaker.appdata.xml
++++ b/utilities/texmaker.appdata.xml
+@@ -21,22 +21,23 @@
+  </description>
+ <screenshots>
+   <screenshot type="default">
+-    <image height="758" width="1280">https://www.xm1math.net/texmaker/assets/img/texmaker_top_linux.png</image>
```
Comment (bilelmoussaoui) [inline appdata.patch]: This patch should go upstream if possible

File: net.xm1math.Texmaker.yaml (hunk 1 above)
Comment (barthalion) [reply net.xm1math.Texmaker.yaml]: It would be safer to remove it than assume it works.

Comment (barthalion) [reply net.xm1math.Texmaker.yaml]: (Going to test it though.)

Comment (barthalion) [reply net.xm1math.Texmaker.yaml]: Bunch of warnings like this:

```
libEGL warning: wayland-egl: could not open /dev/dri/card0 (No such file or directory)
qt.qpa.wayland: Wayland does not support QWindow::requestActivate(
```

But seems to work fine.

File: appdata.patch (hunk 2 above)
Comment (dgcampea) [reply appdata.patch]: Already sent

**Comment by dgcampea** [comment]
> Have you reached upstream if they were interested in (co-)maintaining their app on Flathub?

I did contact them along with the patch for the appdata but I got no response on that matter.

File: net.xm1math.Texmaker.yaml (hunk 1 above)
Comment (nedrichards) [reply net.xm1math.Texmaker.yaml]: At least one of those warnings mean you need to add the `--device=dri` permission.

Comment (dgcampea) [reply net.xm1math.Texmaker.yaml]: done

**Comment by nedrichards** [comment]
bot, build net.xm1math.Texmaker

**Pull request closed without merging**